crypto/src/main/java/org/hyperledger/besu/crypto/SECP256K1.java

```diff
@@ -44,7 +44,7 @@
 
   private static final Logger LOG = LogManager.getLogger();
 
-  private boolean useNative = true;
+  private boolean useNative = false;
```

Comment:
Add a comment on why we set this to false even though using native is the default behaviour.

Reply:
In the end I changed it back to `true`. If not the tests that don't use BesuCommand, would be using the Java implementation instead of the native one. So it would have changed the test behavior. I've added a `disableNative` function instead and call it when the native libraries should be disabled.